px-spinner.html

```diff
@@ -88,7 +88,18 @@
             value: function(){
               return Px.Modernizr.smil;
             }
-          }
+          },
+          /**
+           * This property is to control whether the spinner is rendered as svg or div element.
+           * If useDiv is false, which is by default value, spinner is rendered as svg,
+           * otherwise the spinner is rendered as div.
+           *
+           */
+          useDiv: {
+            type: Boolean,
+            value: false,
+            notify: true,
```

Comment:
I don't think notify needs to be set here. A one-way binding should be enough.

Reply:
Sure, updated.